charts/bulk-scan-sample-app/requirements.yaml

```diff
@@ -3,6 +3,6 @@ dependencies:
     version: ~2.15.0
     repository: '@hmctspublic'
   - name: bulk-scan
-    version: ~0.1.5
+    version: ~0.1.7
```

Comment:
FYI, there's already version 0.1.8, which should fix external (s2s, etc) service URLs